cylc/flow/option_parsers.py

```diff
@@ -155,6 +155,10 @@ def add_std_options(self):
             help="Output developer information and show exception tracebacks.",
             action="store_true", dest="debug",
             default=(os.getenv("CYLC_DEBUG", "false").lower() == "true"))
```

Comment:
This option_parser.py file, on line 42, references the "cylc insert" command that is being removed in this pr.  
Sorry, I can't seem to add it as a suggested change at the correct point as github doesn't seem to let me edit bits of this file that have not already been edited in your pr. Probs me missing something obvious!

Reply:
Roger that, good spotting.